aries-site/src/examples/components/maskedinput/MaskedSizeUnitsExample.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Form, FormField, MaskedInput } from 'grommet';
+
+export const MaskedSizeUnitsExample = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <Form>
+      <FormField
+        label="Size"
```

Comment:
I really like this example, however, it takes me a minute to understand what I am looking at. "What does nnn gb mean?" Once I try to enter information it becomes clear, but it takes me awhile to get there. Would "Storage Size" as a label help shortcut/minimize that learning curve?

Reply:
Good idea.